packages/nextra-theme-docs/src/misc/default.config.js

```diff
@@ -10,6 +10,7 @@ export default {
   prevLinks: true,
   search: true,
   darkMode: true,
+  collapsed: false,
```

Comment:
Thanks for contributing! I personally prefer a more explicit name, such as `defaultMenuCollapsed: false`, what do you think?

Reply:
absolutely, good call!